audbackend/core/backend/base.py

```diff
@@ -25,6 +30,8 @@ def __init__(
         r"""Host path."""
         self.repository = repository
         r"""Repository name."""
+        self.opened = False
+        r"""If backend is opened."""
```

Comment:
Shouldn't it be "is open"?

Reply:
I think I phrased it like this, as I would have also said "is connected".
But "is open" or "was opened" seem to indeed better fit.

I would propose

```suggestion
        r"""If backend is open."""
```

Reply:
If we want to be verbose we could also say "If a connection to the repository has been established."

Reply:
Good idea, I changed it accordingly, and updated the screenshot in the description.

Reply:
Should we then also change the error description from "RuntimeError: if backend was not opened" to "RuntimeError: if no connection to the repository has been established"?

Reply:
I would stay with the current message, as the connection that the user has to call `backend.open()` is more obvious.